main.py

```diff
@@ -1,9 +1,148 @@
-print("Wyświetlenie listy zrobionych zadań: 1")
-print("Wyświetlenie listy zadań do wykonania: 2")
-print("Dodaj zadanie: 3")
-while True:
-    odpowiedz = input('Wybierz menu ')
-    if odpowiedz == "3":
-        nowe_zadanie = input("Napisz jakie zadanie chcesz dodać: ")
-        print(nowe_zadanie)
-        break
\ No newline at end of file
+import sqlite3
+from task import Task
+
+task_list = []
+database = "pythonsqlite.db"
+
+
+def create_connection(db_file):
+    """ create a database connection to the SQLite database
+        specified by db_file
+    :param db_file: database file
+    :return: Connection object or None
+    """
+    conn = None
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except sqlite3.Error as e:
+        print(e)
+
+    return conn
+
+
+def create_table(conn, create_table_sql):
+    """ create a table from the create_table_sql statement
+    :param conn: Connection object
+    :param create_table_sql: a CREATE TABLE statement
+    :return:
+    """
+    try:
+        c = conn.cursor()
+        c.execute(create_table_sql)
+    except sqlite3.Error as e:
+        print(e)
+
+
+def create_task(conn, task_name):
+    """
+    Create a new task
+    :param conn:
+    :param task_name:
+    :return:
+    """
+
+    sql = """ INSERT INTO tasks(name,status)
+              VALUES(?,?) """
+    cur = conn.cursor()
+    cur.execute(sql, task_name)
+    conn.commit()
+    return cur.lastrowid
+
+
+def select_all_tasks(conn):
+    """
+    Query all rows in the tasks table
+    :param conn: the Connection object
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM tasks")
+
+    rows = cur.fetchall()
+
+    for row in rows:
+        print(row)
+
+
+def select_task_by_status(conn, status):
+    """
+    Query tasks by status
+    :param conn: the Connection object
+    :param status:
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM tasks WHERE status=?", (status,))
+
+    rows = cur.fetchall()
+
+    for row in rows:
+        print(row)
+
+
+def main_menu():
+    print("1: Wyświetlenie listy zrobionych zadań")
+    print("2: Wyświetlenie listy zadań do wykonania")
+    print("3: Dodaj zadanie")
+    print("0: Wyjście")
+    choice = input('Wybierz menu: ')
+    if choice == "1":
+        complete_task()
+    elif choice == "2":
+        incomplete_task()
+    elif choice == "3":
+        new_task()
+    elif choice == "0":
+        exit()
+    else:
+        print("Błędny wybór")
+
+
+def return_to_menu():
+    choice = input('Wyjście z programu [T/n]? ')
+    if choice.lower() == "n":
+        main_menu()
+    elif choice.lower() == "t" or len(choice) == 0:
+        exit()
+    else:
+        print("Błędny wybór")
+        return_to_menu()
+
+
+def new_task():
+    print("Podaj nazwę nowego zadania: ")
+    new = Task(input(), False)
```

Comment:
Z bajerów, to możesz dodać sprawdzanie, czy nazwa zadania jest pusta i nie przepuszczać pustego tytułu

Reply:
Dodano